test/custom-source/1-template.yaml

```diff
@@ -28,3 +28,4 @@ spec:
       ]
   template:
     name: dump-build-template
+    kind: BuildTemplate
```

Comment:
Since other tests specify a `template:` without explicitly specifying a `kind:`, it looks like this is unnecessary, and anything besides `kind: ClusterBuildTemplate` will be interpreted to mean `BuildTemplate`, including unspecified.

If we want an unspecified kind to default to `BuildTemplate`, I think that's okay but we should document it at least.

Reply:
Added a comment in the type definition and in the webhook, and will work on a PR for the `buildtemplates` repo to update the README explaining how to use Build Templates